### PR TITLE
Add function and test for obc creation

### PR DESCRIPTION
**Pull request by ebondare**
Add function and test for obc creation, and also make minor changes to PVC creation tests

### CODE REVIEW

File: frontend/packages/ceph-storage-plugin/integration-tests/tests/2-tests/obc.scenario.ts [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import { browser, ExpectedConditions as until } from 'protractor';
+import { appHost } from '@console/internal-integration-tests/protractor.conf';
+import * as crudView from '@console/internal-integration-tests/views/crud.view';
+import {
+  createNewObjectBucketClaim,
+  deleteObjectBucketClaim,
+  goToObjectBucketClaims,
+  obcStatus,
+} from '../../views/obc.view';
+import { NS, CLAIM_STATUS, STORAGE_CLASS_PATTERNS, DEFAULT_BUCKET_CLASS } from '../../utils/consts';
```
Comment (gnehapk) [inline frontend/packages/ceph-storage-plugin/integration-tests/tests/2-tests/obc.scenario.ts]: Alpha imports.

File: frontend/packages/ceph-storage-plugin/integration-tests/tests/2-tests/obc.scenario.ts [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+describe('Test OBC creation with options.', () => {
+  beforeAll(async () => {
+    await browser.get(`${appHost}/`);
```
Comment (gnehapk) [inline frontend/packages/ceph-storage-plugin/integration-tests/tests/2-tests/obc.scenario.ts]: Would be better to move this in a constant.

File: frontend/packages/ceph-storage-plugin/integration-tests/tests/2-tests/obc.scenario.ts [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+  });
+
+  it('Test OBC with default storage class and bucket class is created and gets bound', async () => {
```
Comment (gnehapk) [inline frontend/packages/ceph-storage-plugin/integration-tests/tests/2-tests/obc.scenario.ts]: Statement can be framed better. May be `Test OBC creation with default storage class and bucket class`

File: frontend/packages/ceph-storage-plugin/integration-tests/views/obc.view.ts [hunk 4]
```diff
@@ -0,0 +1,50 @@
+import { $, $$, browser, ExpectedConditions as until } from 'protractor';
+import * as crudView from '@console/internal-integration-tests/views/crud.view';
+import * as sideNavView from '@console/internal-integration-tests/views/sidenav.view';
+import { click } from '@console/shared/src/test-utils/utils';
+import { CLAIM_STATUS } from '../utils/consts';
+import { ObcType, selectItemFromDropdown } from '../utils/helpers';
+
+// create obc
+export const namespaceDropdown = $$('[class="pf-c-dropdown__toggle pf-m-plain"]').get(0);
```
Comment (gnehapk) [inline frontend/packages/ceph-storage-plugin/integration-tests/views/obc.view.ts]: Will this always be the `0`th element?  I will prefer to have the alternative approach as this approach is not very reliable.

Comment (ebondare) [reply frontend/packages/ceph-storage-plugin/integration-tests/views/obc.view.ts]: Thanks, I made the changes in obc and pvc views to get rid of it.

File: frontend/packages/ceph-storage-plugin/integration-tests/tests/2-tests/obc.scenario.ts (hunk 3 above)
Comment (ebondare) [reply frontend/packages/ceph-storage-plugin/integration-tests/tests/2-tests/obc.scenario.ts]: Done

File: frontend/packages/ceph-storage-plugin/integration-tests/tests/2-tests/obc.scenario.ts (hunk 1 above)
Comment (ebondare) [reply frontend/packages/ceph-storage-plugin/integration-tests/tests/2-tests/obc.scenario.ts]: Done

File: frontend/packages/ceph-storage-plugin/integration-tests/tests/2-tests/obc.scenario.ts (hunk 2 above)
Comment (ebondare) [reply frontend/packages/ceph-storage-plugin/integration-tests/tests/2-tests/obc.scenario.ts]: Can we leave it the way it is for this PR and introduce this change in a separate PR? We use `${appHost}/` in a few other places as well.

File: frontend/packages/ceph-storage-plugin/integration-tests/tests/2-tests/obc.scenario.ts [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+
+  it('Test OBC creation with default storage class and bucket class', async () => {
+    const testObc = {
+      name: 'obc-default-values',
+      namespace: NS,
+      storageClass: STORAGE_CLASS_PATTERNS.NOOBAA_FULL_NAME,
+      bucketClass: DEFAULT_BUCKET_CLASS,
+    };
+    await createNewObjectBucketClaim(testObc, true);
+    browser.wait(until.visibilityOf(obcStatus));
```
Comment (rhrazdil) [inline frontend/packages/ceph-storage-plugin/integration-tests/tests/2-tests/obc.scenario.ts]: missing `await` before browser.wait

**Comment by bipuladh** [comment]
Noobaa tests need to go into the Noobaa console plugin folder. 
https://github.com/openshift/console/pull/3582
If you can wait I will be pushing this test ASAP. And maybe you can make changes on top of it.

**Comment by ebondare** [comment]
This functionality has been implemented within noobaa-storage plugin tests.

**Pull request closed without merging**